notebooks/en/rag_with_hugging_face_gemma_elasticsearch.ipynb

```diff
@@ -0,0 +1,6320 @@
+{
```

Comment:
<p>"Setting <code>USE_ELASTICSEARCH_VECTORISATION=True</code> will make the rest of this notebook set up and use ES-vectorization for your data and querying. **Be aware** that this requires your ES deployment to have at least 1 ML node. I recommend setting autoscaling to true on your Cloud deployment in case the model you choose is too big."</p><p> </p><p>If <code>USE_ELASTICSEARCH_VECTORISATION=False</code>, this notebook sets up and uses the locally provided model..."</p><p> </p><p>Move this paragraph to the top to better help users make the decision.</p><p> </p><p>"ES-vectorizing means your clients don't have to implement it, which is the default here. However, if you don't have any ML nodes or your own embedding setup is better/faster,..."</p><p> </p><p>Change the note to the callout</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Great catch + great idea about moving it to the top. Done.</p>

Reply:
<p>Not fully resolved here yet.</p><p> </p><p>"...chose..." should be <code>choose</code></p>

Reply:
<p>👍</p>